File: android/src/main/java/top/huic/tencent_im_plugin/message/entity/OtherMessageEntity.java
package top.huic.tencent_im_plugin.message.entity;

import top.huic.tencent_im_plugin.enums.MessageNodeType;

/**
 * 其它消息实体
 *
 * @author 蒋具宏
 */
public class OtherMessageEntity extends AbstractMessageEntity {
    /**
     * 参数信息
     */
    private String params;

    public OtherMessageEntity() {
        super(MessageNodeType.Other);
    }

    public String getParams() {
        return params;
    }

    public void setParams(String params) {
        this.params = params;
    }
}

File: ios/Classes/message/entity/OtherMessageEntity.swift
//
//  TextMessageEntity.swift
//  tencent_im_plugin
//
//  Created by 蒋具宏 on 2020/3/15.
//  其它消息实体
public class OtherMessageEntity : AbstractMessageEntity{
    /// 消息内容
    var params : String?;
    
    override init() {
        super.init(MessageNodeType.Other);
    }
}
